docs/site/Troubleshooting-guide.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Troubleshooting Guide'
+lang: en
+layout: page
+keywords:
+  LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Troubleshooting
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Troubleshooting-guide.html
+summary: Transactional API usage in LoopBack 4
+---
+
+The following documentation pages can help you troubleshoot problems you
+encountered.
+
+- [Running and debuggin apps](Running-and-debugging-apps.md)
+- [Setting debug strings](Setting-debug-strings.md)
```

Comment:
Do we want the manual process of keeping this list up to date? I am will be one of the first people to forget to update it.

Is it possible to generate this list automatically? I vaguely remember that I saw a markdown page using Jekyll syntax to enumerate the list of child nodes, but I cannot find it now.

Do we actually need to list the pages here at all? I mean we already have a list of child nodes in the left sidebar. Is it perhaps possible to remove this page entirely and keep the "Troubleshooting Guide" item in the sidebar as an expander (a group name) only?

Reply:
> Is it possible to generate this list automatically? I vaguely remember that I saw a markdown page using Jekyll syntax to enumerate the list of child nodes, but I cannot find it now.

Let me find out. 

> Do we actually need to list the pages here at all? I mean we already have a list of child nodes in the left sidebar. Is it perhaps possible to remove this page entirely and keep the "Troubleshooting Guide" item in the sidebar as an expander (a group name) only?

I vaguely remember Jekyll expects each entry in the sidebar to have the `url`, i.e. need a page for each entry. But I can double check.